### PR TITLE
include: zephyr: dsp: Inconsistent macro names changed

### PR DESCRIPTION
**Pull request by rruuaanng**
This is a PR I submitted a long time ago. 
https://github.com/zephyrproject-rtos/zephyr/pull/73122#issuecomment-2123985994

I needed to close it for personal reasons, so I'm sorry that this change has been delayed until now. It seems that this problem has not been solved yet. So I came back.

other header file:
https://github.com/zephyrproject-rtos/zephyr/blob/6f64f77557967a41ef7df429f0a831787bb8418f/include/zephyr/data/jwt.h#L7

https://github.com/zephyrproject-rtos/zephyr/blob/6f64f77557967a41ef7df429f0a831787bb8418f/include/zephyr/crypto/crypto.h#L17

### CODE REVIEW

File: include/zephyr/dsp/basicmath.h [hunk 1]
```diff
@@ -8,8 +8,8 @@
  * @brief Public APIs for DSP basicmath
  */
 
-#ifndef INCLUDE_ZEPHYR_DSP_BASICMATH_H_
-#define INCLUDE_ZEPHYR_DSP_BASICMATH_H_
+#ifndef ZEPHYR_INCLUDE_DSP_BASICMATH_F16_H_
+#define ZEPHYR_INCLUDE_DSP_BASICMATH_F16_H_
```
Comment (stephanosio) [inline include/zephyr/dsp/basicmath.h:12]: This is incorrect.

File: include/zephyr/dsp/basicmath_f16.h [hunk 2]
```diff
@@ -8,8 +8,8 @@
  * @brief Public APIs for DSP basicmath for 16 bit floating point
  */
 
-#ifndef INCLUDE_ZEPHYR_DSP_BASICMATH_F16_H_
-#define INCLUDE_ZEPHYR_DSP_BASICMATH_F16_H_
+#ifndef ZEPHYR_INCLUDE_DSP_BASICMATH_H_
+#define ZEPHYR_INCLUDE_DSP_BASICMATH_H_
```
Comment (stephanosio) [inline include/zephyr/dsp/basicmath_f16.h:12]: same here

**Review by kartben — DISMISSED** [review]

**Comment by kartben** [comment]
@rruuaanng thanks for re-submitting. Comments about addressing the compliance issues reported in CI still apply :) thanks!

**Review by stephanosio — CHANGES_REQUESTED** [review]
Please review the [Contribution Guidelines](https://docs.zephyrproject.org/latest/contribute/guidelines.html#pull-request-guidelines) and update the commit message accordingly.

**Comment by rruuaanng** [comment]
I'm changing my submission. Thanks for the reply

**Comment by rruuaanng** [comment]
I've made changes, Please reviewer again.

**Comment by rruuaanng** [comment]
I'm sorry about the force push, it seems like there was something wrong with my editor, it kept adding spaces. I may need to modify it, please forgive me for this behavior

**Review by kartben — APPROVED** [review]

**Comment by rruuaanng** [comment]
Please forgive me for not being familiar with this. Why does CI appear this
```
 twister-build job result: cancelled
***
Error: The twister-build job did not pass.
Error: Process completed with exit code 1.
```

**Comment by stephanosio** [comment]
> Please forgive me for not being familiar with this. Why does CI appear this

Please open a PR from a fork branch named other than `main`.

**Comment by rruuaanng** [comment]
Thanks for your reply, I will reopen a PR. I'm sorry for this behavior :-(.

**Comment by rruuaanng** [comment]
Thanks for your reply, I will reopen a PR. I'm sorry for this behavior :-(.

**Pull request closed without merging**